extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -139,15 +139,15 @@ class AmpApesterMedia extends AMP.BaseElement {
     this.width_ = getLengthNumeral(width);
     this.height_ = getLengthNumeral(height);
     this.random_ = false;
-    this.mediaAttribute_ = userAssert(
+    this.mediaAttribute_ = /** @type {string} */ (userAssert(
         this.element.getAttribute('data-apester-media-id') ||
-        (this.random_ = this.element.getAttribute(
+        (this.random_ = this.element.hasAttribute(
```

Comment:
Intentional change? From string to boolean?

Reply:
Yes, there was a mismatch earlier, that I discovered via `gulp check-types`.

Reply:
The `playlist` property is only used as a boolean.

Reply:
@aghassemi @jridgewell let me know what the desired fix is here

Reply:
Done in https://github.com/ampproject/amphtml/commit/a0d8a000bb441d47af92effd8a069f16e3675d5f.